src/containers/FormikForm/SlugField.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useRef, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { FieldProps } from 'formik';
+import { TextArea } from '@ndla/forms';
+import FormikField from '../../components/FormikField';
+
+interface Props {
+  maxLength?: number;
+  name?: string;
+  handleSubmit: () => Promise<void>;
+  type?: string;
+}
+
+const SlugField = ({ name = 'slug', handleSubmit }: Props) => {
+  const { t } = useTranslation();
+  const handleSubmitRef = useRef(handleSubmit);
+  useEffect(() => {
+    handleSubmitRef.current = handleSubmit;
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
?

Reply:
Var en kopi fra `TitleField` men du har jo helt rett. Stoler kanskje litt for blindt på koden som er skrevet fra før